packages/theme/src/Components/Markdown/index.tsx

```diff
@@ -1,13 +1,15 @@
 import { markdownTransformIpfsURL } from '@masknet/shared-base'
-import { memo } from 'react'
-import ReactMarkdown, { Options } from 'react-markdown'
-import remarkGfm from 'remark-gfm'
-import rehypeRaw from 'rehype-raw'
-
-interface Props extends Options {}
+import { memo, use, lazy } from 'react'
+import type {} from 'react/next'
+import type { Options } from 'react-markdown'
+export type { Options } from 'react-markdown'
 
+// Note: this 3 dependencies crashes in background script. we need to lazy load it until use.
+const ReactMarkdown = lazy(() => import('react-markdown'))
 // TODO Succeed another Markdown in '@masknet/shared'
-export const Markdown = memo<Props>(({ children, ...props }) => {
+export const Markdown = memo<Options>(function Markdown({ children, ...props }) {
+    const remarkGfm = use(import('remark-gfm').then((x) => x.default))
+    const rehypeRaw = use(import('rehype-raw').then((x) => x.default))
```

Comment:
Can we move them out of the component? Lazy loading inside causes UI to flicker.

You can checkout RSS3 feed cards like this one, and click to open the detail dialog.

<img width="933" alt="image" src="https://user-images.githubusercontent.com/1141198/201506151-be46d153-9e50-41b8-bcaa-bc90907f64d4.png">

Reply:
you can write `import('remark-gfm')` in RSS3 plugin's SNS Adaptor's `init()` method to pre-load it.

Reply:
I don't think preloading would resolve it. The flicker happens every time the dialog opens.